README-Docker.md

```diff
@@ -0,0 +1,35 @@
+# Docker README
+
+A `Dockerfile` has been provided in order to make this code more convenient to run. To get started via `Docker`, you'll need:
+
+* A working Docker environment
+* A `git checkout` of this repository
+* A copy of `GeoLite2-Country.mmdb` in the root of this repository
+
+## Building Your Docker Container
+
+After you have checked out this repository and added `GeoLite2-Country.mmdb` to the root directory of the repository, run this command from the root directory:
+
+```bash
+docker build . -t mmdb-from-go
```

Comment:
I had to run the command using `sudo` here. I am not sure if that is a problem with my environment or if that's just the way it is on Ubuntu.

```
nchelluri@grugbarn:~/dev/mmdb-from-go-blogpost [oalders/dockerfile=] $ docker build . -t mmdb-from-go
ERRO[0010] Can't add file /home/nchelluri/dev/mmdb-from-go-blogpost/.git/hooks/update.sample to tar: io: read/write on closed pipe 
ERRO[0010] Can't close tar writer: io: read/write on closed pipe 
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post http://%2Fvar%2Frun%2Fdocker.sock/v1.40/build?buildargs=%7B%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=Dockerfile&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&session=dvsr7surue6duuflswyad4huu&shmsize=0&t=mmdb-from-go&target=&ulimits=null&version=1: dial unix /var/run/docker.sock: connect: permission denied
```

I am wondering if you can put a comment like "You might have to run this with `sudo` if you see permission errors." after the example command. What do you think?